MapleServer2/MapleServer.cs

```diff
@@ -54,7 +54,7 @@ public static async Task Main()
         }
 
         // Schedule daily reset and repeat every 24 hours
-        TaskScheduler.Instance.ScheduleTask(0, 0, 24, DailyReset);
+        TaskScheduler.Instance.ScheduleTask(0, 0, 24, 24 * 60, DailyReset);
```

Comment:
Have you considered swapping these parameter types out for `TimeSpan`s? I find the parameter list quite confusing - we have a timestamp in its separate parts, then a parameter in minutes. Not sure if this would make it better or worse - just a suggestion.